pr_check.sh

```diff
@@ -1,4 +1,28 @@
-set -exv
+#!/bin/bash
 
-pipenv install -d
-flake8
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+export APP_NAME="ros"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ros"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/repository/cloudservices/ros-backend"
+export DOCKERFILE="Dockerfile"
+export COMPONENTS_W_RESOURCES="vmaas"
```

Comment:
is `vmass` required for ros deployment?

Reply:
@digitronik we don't require that

Reply:
so i guess we can remove 
`export COMPONENTS_W_RESOURCES="vmaas"`

Reply:
@apuntamb FYI